lms/djangoapps/course_api/tests/test_views.py

```diff
@@ -85,8 +85,13 @@ def test_as_inactive_user(self):
         self.verify_response(params={'username': inactive_user.username})
 
     def test_missing_username(self):
+        response_to_anonymous = self.verify_response(expected_status_code=200)
```

Comment:
I didn't mean the call for computing the `response_to_missing_username`.  That is definitely still a required test.

I meant the equivalency test with the response_to_anonymous.  If you tested its equality with a non-anonymous user response, it would still work.